app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class UserMailer < ApplicationMailer
+  def account_activation user
+    @user = user
+    mail to: user.email, subject: t("mail.activation")
+  end
+
+  def password_reset
```

Comment:
nó gửi mail text hay html ấy nhỉ

Reply:
Mail này được gửi theo định dạng html chị ạ

Reply:
c muốn gửi text thì code gì thêm bây h

Reply:
def password_reset user
    @user = user

    mail to: user.email, subject: "Password reset ", 
      body: "Noi dung buc thu
         ..................." 

  end

Reply:
Hoặc trong file environtment.rb thiết lập là
ActionMailer::Base.default :content_type => "text/plain"
nó sẽ gửi nội dung trong file text.erb